components/inputs/input-percent.js

```diff
@@ -169,9 +156,5 @@ class InputPercent extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(Rt
 		}
 	}
 
-	_handleSymbolClick() {
```

Comment:
No longer needed here, but this gets moved to d2l-input-text for the same usability reason.